calico_containers/tests/st/test_base.py

```diff
@@ -44,3 +48,32 @@ def start_etcd(self):
             initial_cluster="calico=http://%s:2380" % self.ip,
             initial_cluster_state="new",
         )
+
+    def assert_powerstrip_up(self, host):
```

Comment:
It's the `docker_host=True` part. That will use port 2377 which will only work if powerstrip is up.